docs/changelog/1422.md

```diff
@@ -1 +1 @@
-- Remove second-order time interpolation (based on previous windows) from Waveform class. Add higher-order BSpline interpolation (based on multiple samples in current window) to Waveform class.
+- Added support for higher-order BSpline interpolation (based on multiple samples in current window).
```

Comment:
```suggestion
- Added support for higher-order B-spline interpolation (based on multiple samples in current window).
```

Because this is how it is written [here](https://en.wikipedia.org/wiki/B-spline)